
<launch>

    <arg name="arm_type" default="" />

    <include file="$(find sandbox_advait_darpa_m3)/launch/occupancy_grid_ode.launch" />

    <node name='contact_memory_node' pkg="sandbox_advait_darpa_m3"
        type="contact_memory.py" output="log" respawn="false"
        args="--$(arg arm_type)">
    </node>

    <node name='log_and_monitor_node' pkg="sandbox_advait_darpa_m3"
        type="log_and_monitor_node.py" output="screen" respawn="false"
        args="--$(arg arm_type)">
    </node>

    <node name='interactive_marker_server' pkg="sandbox_advait_darpa_m3"
        type="interactive_marker_server.py" output="log" respawn="false"
        args="--sim">
    </node>

</launch>


